static/js/main.js

```diff
@@ -61,6 +61,19 @@ var header = {
         var user_id = $("#user_id").val();
         var password = $("#password").val();
         var verify_password = $("#verify_password").val();
+        var email_list_announcement = '0';
+        var email_list_user = '0';
+        var announcement_checked = document.getElementById("email_list_announcement");
+        var user_checked = document.getElementById("email_list_user");
+
+        
+        if (announcement_checked.checked == true) {
+            email_list_announcement = '1';
```

Comment:
Because in models.py I defined the data type of email_list as String. Of course, we can change it to boolean or integer if necessary. '1' means users choose to join the corresponding email_list_anncouncement or email_list_user. '0' means they won't join.

Reply:
1. It should be either Integer or Boolean. I would say Boolean.
2. What is the difference between email_list_anncouncement and email_list_user?

Reply:
1. Ok，I will change to Boolean
2. GraphSpace administrators post announcements to announcement email list. Users use user email list to post messages and discuss questions.